src/main/kotlin/adder/model/Numbers.kt

```diff
@@ -0,0 +1,43 @@
+package adder.model
+
+data class Numbers(val nums: List<Number>) {
+    constructor(input: Collection<String>) : this(input.map(::Number))
+
+    constructor(input: String) : this(split(input))
+
+    fun getSum(): Int {
+        return nums.sumBy { it.value }
+    }
+
+    companion object {
+        private const val REGEX_FOR_DELIMITER = "//(.)\\\\n(.*)"
```

Comment:
수많은 이스케이프가 필요한 문자열의 경우 3중 따옴표 문자열을 사용하면 더 깔끔하게 표현할 수 있습니다.

```suggestion
        private const val REGEX_FOR_DELIMITER =
            """//(.)\\n(.*)"""
```